tests/api2/test_core_download.py

```diff
@@ -0,0 +1,30 @@
+from os import stat
+
+from urllib.request import urlretrieve
+from middlewared.test.integration.utils.client import truenas_server
+from middlewared.test.integration.utils import call
+
+
+def test_get_download_for_config_dot_save():
+    # ping core for sanity
+    assert call('core.ping') == 'pong'
+
+    # set up core download
+    results = call('core.download', 'config.save', [], 'freenas.db')
+    url = results[1]
+    job_id = results[0]
+
+
+    # is the job running?
+    results = call('core.get_jobs', [['id', '=', job_id]], {"get": True})
+    assert results['state'] == 'RUNNING', results.text
```

Comment:
`results` is a dictionary now so it won't have a `text` attribute (same on line 30)